pyproject.toml

```diff
@@ -97,7 +97,13 @@ addopts = """
     --doctest-glob="*.rst" --doctest-glob="*.md" --ignore=docs/examples
     """
 # https://iscinumpy.gitlab.io/post/bound-version-constraints/#watch-for-warnings
-filterwarnings = "error"
+filterwarnings = [
+    "error",
+    "ignore:Use `ophyd_async.epics.core` instead of `ophyd_async.epics.signal` and `pvi`:DeprecationWarning",
+    "ignore:Use `StandardReadableFormat.CONFIG_SIGNAL` instead of `ConfigSignal`:DeprecationWarning",
+    "ignore:Use `StandardReadableFormat.HINTED_SIGNAL` instead of `HintedSignal`:DeprecationWarning",
+    "ignore:Use `StandardReadableFormat.HINTED_UNCACHED_SIGNAL` instead of `HintedSignal.uncached`:DeprecationWarning",
+]
```

Comment:
I think ophyd-async should adopt stricter same standards for ignoring warnings in tests, since tests in dodal and bluesky currently run with strict warnings-are-errors.
If ophyd-async code raises deprecation warnings then it means we have to add similar ignores to our own tests which negates the purpose of adding the deprecation warnings in the first place.

Reply:
Let me see if we can localise those checks to a single test...